pkg/apis/core/v1beta1/types.go

```diff
@@ -205,6 +207,24 @@ type DataVolumeSourceVDDK struct {
 	SecretRef string `json:"secretRef,omitempty"`
 }
 
+// DataVolumeSourceRef defines an indirect reference to the source of data for the DataVolume
+type DataVolumeSourceRef struct {
+	// The kind of the source reference, currently only DataSource is supported
+	Kind DataVolumeSourceRefKind `json:"kind"`
+	// The namespace of the source reference
+	Namespace string `json:"namespace"`
```

Comment:
Maybe make the a pointer and have it optional?  Default is same namespace as the DataVolume

Reply:
sure